WordPress/Classes/Utility/Analytics/WPAnalyticsTrackerMixpanel.m

```diff
@@ -45,10 +45,26 @@ - (instancetype)initWithManagedObjectContext:(NSManagedObjectContext *)context m
         _aggregatedStatProperties = [[NSMutableDictionary alloc] init];
         _mixpanelProxy = mixpanelProxy;
         _context = context;
+
+        [self migrateSessionCountFromProxy: mixpanelProxy];
     }
     return self;
 }
 
+// Migrates the session count away from mixpanel super properties and into
+// Shared defaults.
+- (void)migrateSessionCountFromProxy:(MixpanelProxy *)mixpanelProxy
+{
+    NSInteger sessionCount = [WPAppAnalytics sessionCount];
+    if (sessionCount > 0) {
+        return;
+    }
+    sessionCount = [[mixpanelProxy.currentSuperProperties numberForKey:SessionCount] integerValue];
```

Comment:
A super property is set on every event - so before this PR I think session count is set on every single event (not just app opened). If that's no longer desired (it's probably overkill) I suggest also deleting the super property entirely so it's not written on every other event with the old no-longer-incremented count. If we want to keep it then you'll also want to update the super property every time it's incremented so every event gets it.

Not. confusing. at. all.